app/controllers/users_controller.rb

```diff
@@ -19,7 +19,7 @@ def index
                user_scope.search(params[:search])
              else
                user_scope.order(sort_param => :desc)
-             end.where.not(deleted: true, community_users: { deleted: true })
+             end.where.not(deleted: true).where.not(community_users: { deleted: true })
```

Comment:
Does this no longer work as one clause with the upgrade?

Reply:
It triggers a deprecation warning because rails 6.1 will change the behaviour of having multiple items in one not clause. It changes from `!a && !b` to `!(a && b)` aka `!a || !b`.